pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -2576,6 +2576,21 @@ let
         };
       };
 
+      lapo.asn1js = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "asn1js";
+          publisher = "lapo";
+          version = "0.1.4";
+          hash = "sha256-utbIKlwNHnJZj/51f8hEDmUA/A26De/gY73iT4tXKRU=";
+        };
+        meta = {
+          description = "Decode ASN.1 content inside VSCode";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=lapo.asn1js";
+          homepage = "https://github.com/lapo-luchini/vscode-asn1js";
+          license = lib.licenses.isc;
+        };
```

Comment:
Missing `maintainers` field.

Reply:
Optional: Consider adding yourself as maintainer so you get notified on updates to package and breakage.

Reply:
Does not seem to be documented as requirement in the extensions README.

Reply:
> Does not seem to be documented as requirement in the extensions README.

Not all meta fields are documented there. Will open a PR adding it then.
Listing yourself as maintainer is optional. We highly recommend people to do it, since there is not a better maintainer than an actual user of a package. Which knows when it breaks and how it functions.